packages/apidom-core/check-types-test.tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "allowImportingTsExtensions": true
```

Comment:
Does this need to be here? As we're stipulated, supporting `.js` extension is enabled by default

Reply:
We're using '.ts` and not .`js` imports. So this is fine.